R/lancelot_rt.R

```diff
@@ -456,7 +456,8 @@ lancelot_Rt_mean_duration_weighted_by_infectivity <- function(step, pars) {
   p_C <- combine_steps_groups(
     step, pars$n_groups, n_time_steps,
     n_strains = length(pars$strain_transmission),
-    n_vacc_classes = pars$n_vacc_classes, pars$p_C_step, pars$rel_p_sympt
+    n_vacc_classes = pars$n_vacc_classes, pars$p_C_step, pars$rel_p_sympt,
+    pars$strain_rel_p_sympt
```

Comment:
```suggestion
    n_vacc_classes = pars$n_vacc_classes, p_step = pars$p_C_step, rel_ p = pars$rel_p_sympt,
    strain_rel_p = pars$strain_rel_p_sympt
```

Can you name all arguments following n_vacc_classes please (otherwise there are hard-to-find bugs around argument order)